composer.json

```
@@ -2,6 +2,7 @@
     "require": {
         "php": ">=5",
         "divido/merchant-sdk-guzzle-6": "dev-master",
-        "divido/merchant-sdk": "v2.1.1"
+        "divido/merchant-sdk": "v2.1.1",
+        "symfony/polyfill-intl-idn": "~1.17.1"
     }
```

I had a think about this, and I think it would actually make more sense to use composer's [conflict](https://getcomposer.org/doc/04-schema.md#conflict) property to restrict this.

That way, rather than saying "we need X package at Y version", we are instead saying "we don't want to ever install X package within Y version range". If Guzzle ever dropped the requirement for `symfony/polyfill-intl-idn`, this library would then automatically drop it also, as it's not a requirement.

This should be achievable by doing the following:

```suggestion
        "divido/merchant-sdk": "v2.1.1"
    },
    "conflict": {
        "symfony/polyfill-intl-idn": ">=1.18"
    }
```

Give it a try / let me know what you think!